src/tests/unit/tests/popup/scripts/components/header.test.tsx

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
```

Comment:
Should we also remove the scripts folder from the test tree?
src/tests/unit/tests/popup/**scripts**/components/header.test.tsx

Reply:
Good catch... totally forgot about the tests :(

Reply:
@peterdur Updated